examples/no-striping.cpp

```diff
@@ -12,7 +12,7 @@
 #include <fstream>
 #include <stdexcept>
 
-#include "tests/json.hpp"
+#include <nlohmann/json.hpp>
```

Comment:
I think we can delete `tests/json.hpp` at this point, right?

Reply:
I think you did this already in an earlier PR.